<?xml version="1.0" ?>
<robot xmlns:xacro="http://ros.org/wiki/xacro" name="competition_map">

	<link name="world"/>

	<xacro:include filename="$(find ICRA2023_Quadruped_Competition)/urdf/obstacles/1200x2400_OSB.urdf.xacro"/>

	<xacro:obj_1200x2400_OSB parent="world" handle="1" position="0 0 0" orientation="0.0 0.0 0.0"/>
	
	<gazebo>
		<static>true</static>
	</gazebo>
</robot>